docs/connectors.md

```diff
@@ -0,0 +1,47 @@
+# Conduit Connectors
+
+Connectors are an integral part of Conduit. Conduit ships with a couple of connectors that are built into the service to help developers bootstrap pipelines much more quickly. The built-in connectors include Postgres, File, Kafka and Amazon S3 connectors.
```

Comment:
Here it mentions postgres as a builtin connector, but in the Readme it doesn't. It also doesn't mention the generator connector here.

Reply:
fixed